applications/tari_base_node/src/main.rs

```diff
@@ -25,7 +25,11 @@ mod cli;
 mod consts;
 
 use crate::cli::ConfigBootstrap;
+use config::Config;
 use log::*;
+use tari_common::default_config;
+
+const LOG_TARGET: &str = "base_node::root";
```

Comment:
nit:
```suggestion
const LOG_TARGET: &str = "base_node::main";
```

Reply:
Good suggestion. I may change this in future.